File: applicatie/src/main/java/com/vdzon/administratie/model/Boeking.java
package com.vdzon.administratie.model;


import lombok.*;
import org.mongodb.morphia.annotations.Entity;
import org.mongodb.morphia.annotations.Id;

import java.time.LocalDate;

@Entity("boeking")
public abstract class Boeking {
    public abstract String getUuid();
}

File: model/src/main/java/com/vdzon/administratie/model/boekingen/Boeking.kt
package com.vdzon.administratie.model.boekingen

import com.fasterxml.jackson.annotation.JsonTypeInfo
import java.io.Serializable

@JsonTypeInfo(use=JsonTypeInfo.Id.CLASS, include= JsonTypeInfo.As.PROPERTY, property="className")
abstract class Boeking : Serializable {
    abstract val uuid: String
    abstract val omschrijving: String
}
